tests/test_boards.py

```diff
@@ -695,6 +701,135 @@ def test_snow_pi_initial_value_pwm():
         assert [device.pin for device in board.leds] == pins
         assert all(device.pin.state == 0.5 for device in board.leds)
 
+def test_rgbledboard_bad_init():
+    with pytest.raises(ValueError):
+        RGBLEDBoard()
+    with pytest.raises(ValueError):
+        RGBLEDBoard(2)
+    with pytest.raises(ValueError):
+        RGBLEDBoard(pin=2)
+    with pytest.raises(ValueError):
+        RGBLEDBoard(2, 3, 4)
+
+def test_rgbledboard_bad_init_pwm():
```

Comment:
I am confused by these tests.

`RGBLEDBoard((2, 3, 4))` is expected to raise PinPWMUnsupported because the default for `pwn` is `True` (https://github.com/RPi-Distro/python-gpiozero/pull/631/files#diff-8e73cd872390a78e8e626359ec7c5837R1074)

But why are tests such as `RGBLEDBoard((2, 3, 4), (5, 6, 7))` expected to pass?

Reply:
Because those tests are in the list of tests to use `MockPWMPin`: https://github.com/RPi-Distro/python-gpiozero/blob/70fc6af18b2a2dc07dced4a85c2ba62830f7517b/tests/test_boards.py#L20

Reply:
got ya.

I like the comment above it `# dirty, but it does the job`

Reply:
:)